### PR TITLE
enable and disable SUPPORT_URL with environment variable

### PR DESCRIPTION
**Pull request by isankadn**
Description:

On the MFE learning top menu, the "Help" link is currently displayed by default.

With this change, the "Help" menu will only be visible if the  "SUPPORT URL" variable is not empty.

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @isankadn!  Please note that it may take us up to several weeks or months to complete a review and merge your PR.

Feel free to add as much of the following information to the ticket as you can:
- supporting documentation
- [Open edX discussion forum threads](https://discuss.openedx.org/)
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).


Please let us know once your PR is ready for our review and all tests are green.


:warning: We can't start reviewing your pull request until you've submitted a     [signed contributor agreement](https://openedx.org/cla)     or indicated your institutional affiliation. Please see the [CONTRIBUTING](https://github.com/openedx/edx-platform/blob/master/CONTRIBUTING.rst)     file for more information.     If you've signed an agreement in the past, you may need to re-sign.  See     [The New Home of the Open edX Codebase](https://open.edx.org/blog/the-new-home-of-the-open-edx-codebase/)     for details.

    
Once you've signed the CLA, please allow 1 business day for it to be processed.  After this time, you can re-run the     CLA check by editing the PR title.  If the problem persists, you can tag the `@openedx/cla-problems` team in a     comment on your PR for further assistance.

**Review by arbrandes — CHANGES_REQUESTED** [review]
Thanks @isankadn!  I haven't actually tested this yet, but in the meantime, can you please:

* Add SUPPORT_URL to the `.env*` files
* Add an explanation of what this does to this repo's README?

**Comment by isankadn** [comment]
@arbrandes  Thank you very much!
Both items are now completed!

**Comment by DeanJayMathew** [comment]
Thanks for helping @arbrandes !

**Comment by arbrandes** [comment]
@isankadn, thanks!

However, it looks like we're failing some tests.  Can you look into it?

Also, please refer to the automated message above: we need you to sign the CLA before accepting your code.  Mind doing that?  Thanks!

**Comment by isankadn** [comment]
@arbrandes Thank you for your comment, sure I can sign. and the same time, I'm representing ABC for this, I think @DeanJayMathew has already taken care of that. 

and I will look into this error as well. thx. 

> Also, please refer to the automated message above: we need you to sign the CLA before accepting your code. Mind doing that? Thanks!

**Comment by isankadn** [comment]
@arbrandes Thank you for your comment, sure I can sign. and the same time, I'm representing ABC for this, I think @DeanJayMathew has already taken care of that. 

and I will look into this error as well. thx. 

> Also, please refer to the automated message above: we need you to sign the CLA before accepting your code. Mind doing that? Thanks!

**Comment by isankadn** [comment]
@arbrandes fixed the linting issue. and pushed.

**Comment by mphilbrick211** [comment]
Hi @isankadn! Just following up on this PR. In order to push it through, you'll need to complete a [CLA form](https://openedx.atlassian.net/wiki/spaces/COMM/pages/941457737/How+to+start+contributing+to+the+Open+edX+code+base). Please let me know if you have any questions.

**Comment by mphilbrick211** [comment]
Hi @isankadn! Tests have been enabled for this pull request, though you'll still need to fill out a [CLA form](https://openedx.atlassian.net/wiki/spaces/COMM/pages/941457737/How+to+start+contributing+to+the+Open+edX+code+base). Please let us know if you plan to pursue this pull request. Thanks!

**Comment by DeanJayMathew** [comment]
Hi @Michelle Philbrick ***@***.***>

Thank you for following up on this.

@Isanka Wijerathne ***@***.***> (
https://github.com/isankadn, ***@***.***) should be covered under
the organisational contributors agreement I signed on behalf of ABC ONLINE
COURSES LIMITED.

Dean Jay Mathew
Director, ABC Online Courses Limited

Trusted Open edX® Service Providers since 2015
Schedule a meeting with me: https://calendly.com/abconlinecourses
View Our Website <http://www.abconlinecourses.com>
Tel.: +441225293341
Registered Address: Chestnut House
***@***.***,-2.1459131,15z/data=!3m1!4b1!4m8!1m2!2m1!1schestnut+house+53d+beanacre+sn127py!3m4!1s0x48717c9105a9d3d7:0x15804f821ad57207!8m2!3d51.3980641!4d-2.1374675>
(Wiltshire, England, UK)



*Legal Disclaimer: **This email and any files transmitted with it are
confidential and intended solely for the use of the individual or entity to
whom they are addressed and intended. If you have received this email in
error please notify the sender or system manager immediately. This message
contains confidential information and is intended only for the individual
named. If you are not the named addressee you should not disseminate,
distribute or copy this e-mail. Please notify the sender immediately by
e-mail if you have received this e-mail by mistake and delete this e-mail
from your system. If you are not the intended recipient you are notified
that disclosing, copying, distributing or taking any action in reliance on
the contents of this information is strictly prohibited.*

**Comment by mphilbrick211** [comment]
Thanks @DeanJayMathew! I'm not seeing @isankadn listed on our end. Let me follow-up internally to make sure I'm not missing anything.

**Comment by e0d** [comment]
@mphilbrick211 I see him in the CLA file now.

@isankadn I also notice there are some commit lint issues, we use conventional commits as defined here: https://open-edx-proposals.readthedocs.io/en/latest/best-practices/oep-0051-bp-conventional-commits.html#type

**Comment by mphilbrick211** [comment]
Hi @isankadn! Just flagging Ed's comment and Adolfo's requested changes. Also, your CLA should be all set now, so once tests are re-run the check should turn green.

File: src/learning-header/AuthenticatedUserDropdown.jsx [hunk 1]
```diff
@@ -16,10 +16,12 @@ const AuthenticatedUserDropdown = ({ intl, username }) => {
       {intl.formatMessage(messages.dashboard)}
     </Dropdown.Item>
   );
-
+  const SupportUrl = (
+    <a className="text-gray-700 mr-3" href={`${getConfig().SUPPORT_URL}`}>{intl.formatMessage(messages.help)}</a>
+  );
```
Comment (arbrandes) [inline src/learning-header/AuthenticatedUserDropdown.jsx:21]: I don't think we need to specify a separate variable for this.  It's fine if we just do it  below, via:

```
{getConfig().SUPPORT_URL && (
  <a>...</a>
)}

**Review by arbrandes — CHANGES_REQUESTED** [review]

**Comment by arbrandes** [comment]
@isankadn after you address my request, can you please squash your commits into a single one with a conventional commit title?  In this case it would be a `feat:`.

**Comment by mphilbrick211** [comment]
Hi @isankadn! Just checking in on this to see if you've been able to address Adolfo's feedback. Please let us know when this is ready to review. Thank you!

**Comment by mphilbrick211** [comment]
> Hi @isankadn! Just checking in on this to see if you've been able to address Adolfo's feedback. Please let us know when this is ready to review. Thank you!

Hi @isankadn! Just checking in to see if you are able to re-run the failing tests and confirm if the requested changes have been made. Thanks!

**Comment by mphilbrick211** [comment]
> > Hi @isankadn! Just checking in on this to see if you've been able to address Adolfo's feedback. Please let us know when this is ready to review. Thank you!
> 
> Hi @isankadn! Just checking in to see if you are able to re-run the failing tests and confirm if the requested changes have been made. Thanks!

Hi @isankadn friendly ping on this. Are you planning to pursue this PR?

**Comment by mphilbrick211** [comment]
Hi @isankadn - just checking in to see if you plan to pursue this pull request - please let us know. Thanks!

**Comment by mphilbrick211** [comment]
Hi @isankadn - I'm going to close this pull request for now. We can reopen if you'd like to pursue in the future. Thanks!

**Comment by openedx-webhooks** [comment]
@isankadn Even though your pull request wasn’t merged, please take a moment  to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=openedx/frontend-component-header&entry.867055334=https://github.com/openedx/frontend-component-header/pull/276&entry.1484655318=https://github.com/isankadn&entry.752974735=2022-11-05+17:22&entry.1917517419=2023-04-25+00:29&entry.2133058324=No) so we can improve your experience in the future.

**Pull request closed without merging**